llm/src/text_data.py

```diff
@@ -161,6 +164,7 @@ def build_text_dataloader(cfg: DictConfig, device_batch_size: int):
         tokenizer_name=cfg.dataset.tokenizer_name,
         max_seq_len=cfg.dataset.max_seq_len,
         group_method=cfg.dataset.group_method,
+        keep_zip=cfg.dataset.get('keep_zip', False),
```

Comment:
This makes me squint a bit as we're adding a default for this one field but not the others... but it also doesn't seem worth upgrading all the YAMLs just for this. There must be some kind of dataclass - based defaulting thing we could do in the future, but not worth it rn..